src/App.css

```diff
@@ -1,5 +1,6 @@
 .App {
   text-align: center;
   width: 100vw;
-  min-height: 100vh;
+  min-height: 90vh;
+  background-color: white;
```

Comment:
I don't think this change is needed. You can set the background color when you make the styles in JSX.

Reply:
Ok I believe this change was done when you implemented the theme design

Reply:
It was not. [`App.css`](https://github.com/beyondhb1079/s4us/blob/main/src/App.css) in the `main` branch has the old styling.

Reply:
Ok so I just removed line 4&5 from the `App.css `